crates/anvil/server/src/config.rs

```diff
@@ -23,6 +23,13 @@ pub struct ServerConfig {
         arg(long, help = "Disable CORS", conflicts_with = "allow_origin")
     )]
     pub no_cors: bool,
+
+    /// Whether to disable the axum default body size limit
+    #[cfg_attr(
+        feature = "clap",
+        arg(long, help = "Disable the default request body size limit")
```

Comment:
could add a note here that it's 2mb

Reply:
Thats fine, it can say "at the time of writing is 2mb"

Reply:
Updated